CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## UNRELEASED
 
+## [1.1.0-beta1] - 2019-09-09
+- Added BatchConsumer.
```

Comment:
For the PR, let's leave this under UNRELEASED and then bump the version afterwards.